src/main/java/com/liefery/android/stop_badge/StopBadgeFactory.java

```diff
@@ -0,0 +1,37 @@
+package com.liefery.android.stop_badge;
+
+import android.annotation.SuppressLint;
+import android.graphics.*;
+import android.util.LruCache;
+
+@SuppressLint( "NewApi" )
```

Comment:
You already increased the minSdkVersion. Is there still a warning left? May this cause runtime issues?